examples/vue-3-cli/src/stories/OverrideArgs.stories.js

```diff
@@ -0,0 +1,42 @@
+import { h } from 'vue';
+import OverrideArgs from './OverrideArgs.vue';
+
+// Emulate something that isn't serializable
+const icons = {
+  Primary: {
+    template: '<span>Primary Icon</span>',
+  },
+  Secondary: {
+    template: '<span>Secondary Icon</span>',
+  },
+};
+
+export default {
+  title: 'Example/Override Args',
+  component: OverrideArgs,
+  argTypes: {
+    icon: {
+      control: {
+        type: 'select',
+        options: Object.keys(icons),
+      },
+      defaultValue: 'Primary',
+    },
+  },
+};
+
+const Template = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    setup(props) {
```

Comment:
@phated The setup function from the comment in the original issue seems more idiomatic. WDYT?

```
    setup() {
      return { icons };
    },
```

Reply:
@shilman I force pushed that change, though I don't like setting all the icons as props, so I mapped the `icon` prop to the component. I added some additional comments to the examples.